core/src/test/java/org/mskcc/oncotree/utils/OncotreeTestConfig.java

```diff
@@ -174,6 +174,7 @@ private Version setupMockVersion() {
         List<OncoTreeNode> tmpOncoTreeRepositoryMockResponse = new ArrayList<>();
         for (int pos = 0; pos < rawTestValueSource.length; pos = pos + valuesPerCase) {
             OncoTreeNode nextNode = new OncoTreeNode();
+            nextNode.setURI("TESTURI");
```

Comment:
Was this just an oversight during the construction of our unit tests?  I guess we need this because we are now calling getURI() and .endsWith() and would get a NPE if not set -- ok, understood.